{"last_activity_date": 1477095318, "owner": {"display_name": "astromme", "reputation": 103, "profile_image": "https://www.gravatar.com/avatar/84dd3640fe566a329afec588448da697?s=128&d=identicon&r=PG", "user_id": 1004331, "user_type": "registered", "accept_rate": 58, "link": "http://stackoverflow.com/users/1004331/astromme"}, "is_answered": false, "answer_count": 1, "creation_date": 1477093624, "score": 0, "title": "Why does my dragonmapper installation fail with AttributeError for the &#39;hanzi&#39; object?", "tags": ["python"], "view_count": 4, "question_id": 40186867, "link": "http://stackoverflow.com/questions/40186867/why-does-my-dragonmapper-installation-fail-with-attributeerror-for-the-hanzi-o"}